lib/src/install.rs

```diff
@@ -1502,9 +1514,16 @@ async fn install_to_filesystem_impl(state: &State, rootfs: &mut RootSetup) -> Re
             &imgstore,
         )
         .await?;
+
+        if matches!(cleanup, Cleanup::TriggerOnNextBoot) {
+            let sysroot_dir = crate::utils::sysroot_dir(&sysroot)?;
```

Comment:
This is fine as is, really. It's just interesting to note that it's equivalent to `rootfs.physical_root` and I think that one's clearer. would probably make sense to try to port more things to it.